### PR TITLE
[feature] allow toggle sequential download

### PR DESCRIPTION
**Pull request by slikie**
#102

### CODE REVIEW

**Comment by CzBiX** [comment]
Since the code always turns sequential download on, it should be named as 'turn on seq donwload', or add the toggle feature.

**Comment by slikie** [comment]
It is in fact a toggle.

File: src/Api.ts [hunk 1]
```diff
@@ -157,6 +157,10 @@ class Api {
     return this.actionTorrents('setForceStart', hashes, { value: 'true' });
   }
 
+  public toggleSequentialTorrents(hashes: string[]) {
+    return this.actionTorrents('toggleSequentialDownload', hashes, { value: 'true' });
```
Comment (CzBiX) [inline src/Api.ts:161]:
```suggestion
    return this.actionTorrents('toggleSequentialDownload', hashes);
```

**Comment by CzBiX** [comment]
Oh, my mistake.

**Pull request merged**